draft-ietf-acme-acme.md

```diff
@@ -1259,7 +1263,9 @@ Link: <https://example.com/acme/some-directory>;rel="index"
   "contact": [
     "mailto:cert-admin@example.com",
     "mailto:admin@example.com"
-  ]
+  ],
+
+  "orders": "https://example.com/acme/orders/acct/1"
```

Comment:
In the `Account objects` section we use a slightly different form for the `orders` URL which probably makes slightly more sense: `"orders": "https://example.com/acme/acct/1/orders"` (i.e. list of orders for acct `1`). Not that important but they should probably be synced.

Reply:
Good catch, I agree that this format makes more sense and being consistent is the best path forward. Fixed in 17f466a